docs/serving/samples/cloudevents/cloudevents-spring/README.md

```diff
@@ -0,0 +1,139 @@
+# Spring + CloudEvents + Knative
+
+A simple web app written in Java using Spring Cloud Function that can receive CloudEvents. It
+supports running in two modes:
+
+1. The default mode has the app reply to your input events with the output
+   event, which is simplest for demonstrating things working in isolation, but
+   is also the model for working for the Knative Eventing `Broker` concept. The
```

Comment:
```suggestion
   is also the model for working with the Knative Eventing `Broker` concept. The
```

Not sure if it should be "the model of working for the Knative Eventing Broker" - for working for sounds a little bit off, but I'm not sure what would make it clear. 

Maybe even "but is also the model for the Knative Evenging `Broker` concept. Could drop "for working"